packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -416,3 +416,65 @@ export function getFocusableTreeWalker(root: HTMLElement, opts?: FocusManagerOpt
 
   return walker;
 }
+
+/**
+ * Creates a FocusManager object that can be used to move focus within an element.
+ */
+export function createFocusManager(ref: RefObject<HTMLElement>): FocusManager {
```

Comment:
Because it doesn't do anything that requires it to be a hook (i.e. doesn't use any hooks within it). This makes it usable in other contexts than just React components.